src/lang/english.txt

```diff
@@ -2118,19 +2119,46 @@ STR_NETWORK_CONNECTION_DISCONNECT                               :{BLACK}Disconne
 
 STR_NETWORK_NEED_GAME_PASSWORD_CAPTION                          :{WHITE}Server is protected. Enter password
 STR_NETWORK_NEED_COMPANY_PASSWORD_CAPTION                       :{WHITE}Company is protected. Enter password
-STR_NETWORK_COMPANY_LIST_CLIENT_LIST_CAPTION                    :{WHITE}Client list
 
 # Network company list added strings
-STR_NETWORK_COMPANY_LIST_CLIENT_LIST                            :Client list
-STR_NETWORK_COMPANY_LIST_SPECTATE                               :Spectate
-STR_NETWORK_COMPANY_LIST_NEW_COMPANY                            :New company
+STR_NETWORK_COMPANY_LIST_CLIENT_LIST                            :Players Online
```

Comment:
Shouldn't it be "Online players", or "{NUM} players online"? Again, I'm not native so not sure.

Reply:
I like `{NUM} online players`, but I cannot find a way to add the number there .. so `Online players` it is for now.